builder/vsphere/common/step_boot_command.go

```diff
@@ -7,20 +7,25 @@ package common
 import (
 	"context"
 	"fmt"
+	"log"
+	"net"
 	"time"
 
 	"github.com/hashicorp/packer-plugin-sdk/bootcommand"
 	"github.com/hashicorp/packer-plugin-sdk/multistep"
 	packersdk "github.com/hashicorp/packer-plugin-sdk/packer"
 	"github.com/hashicorp/packer-plugin-sdk/template/interpolate"
 	"github.com/hashicorp/packer-plugin-vsphere/builder/vsphere/driver"
+	"github.com/pkg/errors"
 	"golang.org/x/mobile/event/key"
 )
 
+// DefaultBindAddress defines the default IP address for the HTTP server.
+const DefaultBindAddress = "0.0.0.0"
```

Comment:
I may move this to `builder/vsphere/common/http_address.go` as `DefaultHttpBindAddress`.